<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_new_supplied_outlines</class>
 <widget class="QFrame" name="f_new_supplied_outlines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Supplied Outlines</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lb_outlines">
     <property name="text">
      <string>Bulk Outlines Layer:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsMapLayerComboBox" name="ml_outlines_layer"/>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="rad_external_source">
     <property name="text">
      <string>External Source Exists</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lb_external_id">
     <property name="text">
      <string>External ID Field:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QgsFieldComboBox" name="fcb_external_id"/>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lb_data_desc">
     <property name="text">
      <string>Data Description:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="le_data_description"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lb_organisation">
     <property name="text">
      <string>Organisation:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="cmb_organisation"/>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lb_capture_method">
     <property name="text">
      <string>Capture Method:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="cmb_capture_method"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lb_capture_src_grp">
     <property name="text">
      <string>Capture Source Group:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="cmb_capture_src_grp"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lb_external_id_2">
     <property name="text">
      <string>External ID:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QComboBox" name="cmb_external_id"/>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lb_imagery">
     <property name="text">
      <string>Imagery Layer:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QgsMapLayerComboBox" name="mcb_imagery_layer"/>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lb_imagery_field">
     <property name="text">
      <string>Imagery Field:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QgsFieldComboBox" name="fcb_imagery_field"/>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lb_imagery_value">
     <property name="text">
      <string>Imagery Value:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QComboBox" name="cmb_imagery"/>
   </item>
   <item row="16" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="25" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QPushButton" name="btn_ok">
     <property name="maximumSize">
      <size>
       <width>85</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QPushButton" name="btn_cancel">
     <property name="maximumSize">
      <size>
       <width>85</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ml_outlines_layer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
